lib/message.rb

```diff
@@ -0,0 +1,41 @@
+require 'yaml'
+require 'twilio-ruby'
+require 'dotenv'
+
+class SMS
+  def initialize(amount)
+    @amount = amount
+    connect_to_twilio_client
+  end
+
+  def send_sms
+    triger_twilio_sms
+    prints_error_if_any
+  end
+
+  private
+
+  def prints_error_if_any
+    rescue Twilio::REST::RequestError => e
+    puts e.message
+  end
```

Comment:
Actually it does work! =) (just checked it by changing from_number, and got Twilio::REST::RequestError: The From phone number +4411332062011 is not a valid, SMS-capable inbound phone number or short code for your account.)